jp2_pc/Source/Trespass/supportfn.cpp

```diff
@@ -817,9 +817,12 @@ void MyBlt(LPBYTE pbSrc,
     }
     else
     {
+        //Use smaller size when buffer sizes do not match
+        size_t copysize = iSrcBytes == iDstBytes ? iWidthBytes : iWidth * std::min(iDstBytes, iSrcBytes);
+        
         for (y = 0; y < iHeight; y++)
         {
-            memcpy(pbDst, pbSrc, iWidthBytes);
+            memcpy(pbDst, pbSrc, copysize);
             pbDst += iDstPitch;
             pbSrc += iSrcPitch;
         }
```

Comment:
In 32bit color depth mode, the buffer sizes do not match and we would get a crash when opening the pause menu.